### PR TITLE
PBM-617 Described the ability to restore from any backup

### PR DESCRIPTION
**Pull request by nastena1606**
modified:   source/pbm-commands.rst
	modified:   source/point-in-time-recovery.rst
	modified:   source/running.rst

### CODE REVIEW

File: doc/source/point-in-time-recovery.rst [hunk 1]
```diff
@@ -76,16 +82,20 @@ Re-enable |PITR| to resume saving oplog slices:
...
+
+To restore from any backup snapshot, |PBM| requires continuous oplog. After the backup snapshot is made and |PITR| is re-enabled, it copies the oplog saved with the backup snapshot and creates oplog slices from the end time of the latest slice to the new starting point thus making the oplog continuous.
+
 
 .. rubric:: Delete a backup
 
-When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot
-2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
+When you :ref:`delete a backup <pbm.backup.delete>`, all oplog slices that relate to this backup will be deleted too. For example, you delete a backup snapshot 2020-07-24T18:13:09 while there is another snapshot 2020-08-05T04:27:55 created after it.  |pbm-agent| deletes only oplog slices that relate to 2020-07-24T18:13:09.
```
Comment (sandraromanchenko) [inline doc/source/point-in-time-recovery.rst:92]: This part is modified entirely within PBM-652, is this change still necessary?

File: doc/source/point-in-time-recovery.rst [hunk 2]
```diff
@@ -56,9 +56,15 @@ Run |pbm-restore| and specify the timestamp from the valid range:
...
-   $ pbm restore --time="2020-07-14T14:27:04"
+   $ pbm restore --time="2020-12-14T14:27:04"
 
-Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of it (the time ranges in PITR section of ``pbm list`` output must be later then the selected backup snapshot). 
+Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of this backup. The time stamp you specify for the restore must meet the following conditions:
+
+* It must be from the time ranges in the PITR section of ``pbm list`` output
+* It must be later than the backup snapshot used for the restore. 
+
+To illustrate this behavior, let’s use the ``pbm list`` output from the previous example. For timestamp ``2020-07-14T14:27:04``,
```
Comment (sandraromanchenko) [inline doc/source/point-in-time-recovery.rst:66]: Pls correct the date 2020-07-14 to 2020-12-14

**Review by sandraromanchenko — CHANGES_REQUESTED** [review]

File: doc/source/point-in-time-recovery.rst (hunk 1 above)
Comment (nastena1606) [reply doc/source/point-in-time-recovery.rst:92]: I've reflected these changes in https://github.com/percona/percona-backup-mongodb/pull/568 so after the merge the doc will be consistent

File: doc/source/point-in-time-recovery.rst (hunk 2 above)
Comment (nastena1606) [reply doc/source/point-in-time-recovery.rst]: Thanks. Fixed.

**Review by sandraromanchenko — APPROVED** [review]

File: doc/source/point-in-time-recovery.rst [hunk 3]
```diff
@@ -56,9 +58,15 @@ Run |pbm-restore| and specify the timestamp from the valid range:
 
 .. code-block:: bash
 
-   $ pbm restore --time="2020-07-14T14:27:04"
+   $ pbm restore --time="2020-12-14T14:27:04"
+
+Restoring to the point in time requires both a backup snapshot and oplog slices that can be replayed on top of this backup. The time stamp you specify for the restore must meet the following conditions:
+
+* It must be from the time ranges in the PITR section of ``pbm list`` output
```
Comment (dAdAbird) [inline doc/source/point-in-time-recovery.rst:65]:
```suggestion
* It must be within the time ranges in the PITR section of ``pbm list`` output
```

File: doc/source/point-in-time-recovery.rst [hunk 4]
```diff
@@ -56,9 +58,15 @@ Run |pbm-restore| and specify the timestamp from the valid range:
...
+* It must be later than the backup snapshot used for the restore.
```
Comment (dAdAbird) [inline doc/source/point-in-time-recovery.rst:66]: _If `--base-snapshot` is set. Otherwise, PBM will choose the base on it's own and the user doesn't have think about that - just follow the first condition_

File: doc/source/point-in-time-recovery.rst (hunk 1 above)
Comment (dAdAbird) [reply doc/source/point-in-time-recovery.rst:94]: Still, I don't think we should have this paragraph anymore. As of v1.6 `pbm delete-backup` doesn't delete any pitr-chunks in any case.

**Review by dAdAbird — CHANGES_REQUESTED** [review]

File: doc/source/point-in-time-recovery.rst (hunk 4 above)
Comment (nastena1606) [reply doc/source/point-in-time-recovery.rst:66]: Won't this auto-selected snapshot be earlier in time than the specified PITR timestamp?

Comment (dAdAbird) [reply doc/source/point-in-time-recovery.rst:66]: It will. I just meant that if the user doesn't set `--base-snapshot` they shouldn't think about any snapshot. Any time that follows the first condition implicitly has some backup that finished before this time and pbm will choose that backup automatically. So the user doesn't have to worry about any snapshot in such a case.

Maybe
```
If `--base-snapshot` is used the timestamp must be later than the backup snapshot used for the restore.
```

Comment (nastena1606) [reply doc/source/point-in-time-recovery.rst]: Rewritten

**Review by dAdAbird — APPROVED** [review]
